Content.Client/Audio/Jukebox/JukeboxBoundUserInterface.cs

```diff
@@ -114,5 +124,16 @@ protected override void Dispose(bool disposing)
         _menu.Dispose();
         _menu = null;
     }
+
+    protected override void UpdateState(BoundUserInterfaceState state)
+    {
+        var castState = (JukeboxBoundUserInterfaceState) state;
+
+        if (_menu == null)
+            return;
+
+        _menu.PopulateQueue(castState.SongIdQueue);
+        _menu.SetIsPlaying(castState.IsPlaying);
+    }
 }
```

Comment:
Put this on the componentstate instead please.

Reply:
Done